libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -838,6 +852,29 @@ class MapboxNavigation @VisibleForTesting internal constructor(
             callback,
         )
     }
+    
+    @ExperimentalPreviewMapboxNavigationAPI
+    fun setPreviewRoutes(routes: List<NavigationRoute>) {
```

Comment:
```suggestion
    fun setRoutesPreview(routes: List<NavigationRoute>) {
```

I'm fine with either, but everything should be called `RoutesPreview` or `PreviewRoutes`. That way we can search for the word and find all the objects and functions